Frends.AmazonS3.DownloadObject/Frends.AmazonS3.DownloadObject/Definitions/Input.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AmazonS3.DownloadObject
```

Comment:
Please use one-line namespace definition. Also, the namespace should be `Frends.AmazonS3.DownloadObject.Definitions`.